llvm/lib/Target/RISCV/VentusInstrFormatsV.td

```diff
@@ -62,6 +62,27 @@ class RVInstSetVLi<dag outs, dag ins, string opcodestr, string argstr>
   let Opcode = OPC_OP_V.Value;
 }
 
+// VENTUS VOP VI instructions with 11 bits immediate number
+class PseudoVOPVIIMM11<SDPatternOperator Op> :
+        Pseudo<(outs VGPR:$vd), (ins VGPR:$vs2, simm11:$imm11),
+        [(set (XLenVT VGPR:$vd), (Op (XLenVT VGPR:$vs2), simm11:$imm11))]> {
+  // let usesCustomInserter = 1;
+  let hasSideEffects = 1;
+  let mayLoad = 0;
+  let mayStore = 0;
+  let IsVOPIMM11 = 1;
+}
+
+class PseudoVOPIVIMM11<SDPatternOperator Op> :
+        Pseudo<(outs VGPR:$vd), (ins VGPR:$vs2, simm11:$imm11),
+        [(set (XLenVT VGPR:$vd), (Op simm11:$imm11, (XLenVT VGPR:$vs2)))]> {
+  // let usesCustomInserter = 1;
+  let hasSideEffects = 1;
+  let mayLoad = 0;
+  let mayStore = 0;
+  let IsVOPIMM11 = 1;
```

Comment:
let hasSideEffects = 1, mayLoad = 0, mayStore = 0, IsVOPIMM11 = 1 in {
class PseudoVOPVIIMM11<SDPatternOperator Op> :
        Pseudo<(outs VGPR:$vd), (ins VGPR:$vs2, simm11:$imm11),
        [(set (XLenVT VGPR:$vd), (Op (XLenVT VGPR:$vs2), simm11:$imm11))]>;
class PseudoVOPIVIMM11<SDPatternOperator Op> :
        Pseudo<(outs VGPR:$vd), (ins VGPR:$vs2, simm11:$imm11),
        [(set (XLenVT VGPR:$vd), (Op simm11:$imm11, (XLenVT VGPR:$vs2)))]>

}